sigs.yml

```
@@ -0,0 +1,502 @@
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Python: SDK'
+    meeting: Every Thursday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs
+    chat:
+      - type: slack
+        name: '#otel-python'
+        id: C01PD4HUVBL
+    invites: calendar-python
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Ruby: SDK'
+    meeting: Every Tuesday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE
+    chat:
+      - type: slack
+        name: '#otel-ruby'
+        id: C01NWKKMKMY
+      - type: other
+        name: GitHub Discussions
+        link: https://github.com/open-telemetry/opentelemetry-ruby/discussions
+    invites: calendar-ruby
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Rust: SDK'
+    meeting: Every Tuesday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4
+    chat:
+      - type: slack
+        name: '#otel-rust'
+        id: C03GDP0H023
+      - type: other
+        name: Gitter
+        link: https://gitter.im/open-telemetry/opentelemetry-rust
+    invites: calendar-rust
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Swift: SDK'
+    meeting: Every Thursday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1LugL8r4bAkbTxZ1Gq_6j_8SDv1LuZaNeUdR-QLN4d48
+    chat:
+      - type: slack
+        name: '#otel-swift'
+        id: C01NCHR19SB
+    invites: calendar-swift
+    tcSponsors: []
+    gcLiaison: []
+  - name: Communications (Website, [Documentation](https://opentelemetry.io/docs/),
+      etc.)
+    meeting: Every other week on Monday at 10:00 PT
+    notes:
+      type: gDoc
+      value: 1wW0jLldwXN8Nptq2xmgETGbGn9eWP8fitvD5njM-xZY
+    chat:
+      - type: slack
+        name: '#otel-comms'
+        id: C02UN96HZH6
+    invites: calendar-comms
+    tcSponsors: []
+    gcLiaison:
+    - name: 'Severin Neumann'
+      github: svrnm
+  - name: End-User WG
+    meeting: Every other week on Thursday at 10:00 PT
+    notes:
+      type: gDoc
+      value: 1e-UNZA3Tuno9b53RQbe--whUcO0VIXF3P81oXsrBK6g
+    chat:
+      - type: slack
+        name: '#otel-user-research'
+        id: C01RT3MSWGZ
+    invites: calendar-user-research
+    tcSponsors: []
+    gcLiaison:
+    - name: Daniel Gomez Blanco
+      github: danielgblanco
+  - name: eBPF
+    meeting: Every week on Tuesday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 13GK915hdDQ9sUYzUIWi4pOfJK68EE935ugutUgL3yOw
+    chat:
+      - type: slack
+        name: '#otel-ebpf'
+        id: C02AB15583A
+    invites: calendar-ebpf
+    tcSponsors: []
+    gcLiaison: []
+  - name: Agent Management WG
+    meeting: Every other week on Tuesday at 11:00 PT
+    notes:
+      type: gDoc
+      value: 19WA5-ex8rNFIBIyVb5VqMXfWNmUQwppGhN8zBeNG0f4
+    chat:
+      - type: slack
+        name: '#otel-agentmanwg'
+        id: C02J58HR58R
+    invites: calendar-agent-mgmt
+    tcSponsors: []
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG